src/lib.rs

```diff
@@ -61,69 +76,253 @@ impl InlinerBuilder {
         self
     }
 
-    /// Configures whether unexpanded modules (due to a missing file) will lead
-    /// to an `Err` return value or not.
+    /// Configures whether unexpanded modules (due to missing files or invalid Rust source code)
+    /// will lead to an `Err` return value or not.
     ///
     /// Default: `false`.
-    pub fn error_not_found(&mut self, error_not_found: bool) -> &mut Self {
-        self.error_not_found = error_not_found;
+    pub fn track_errors(&mut self, track_errors: bool) -> &mut Self {
+        self.track_errors = track_errors;
         self
     }
```

Comment:
Question -- now that we can pull the original `syn::File` out of the error type in case of incomplete inlining, do we even need this? Or can we just always have this set to `true`, remove the API, and ask users to pull it out (or use the simple `parse_and_inline_modules` interface) if they wish.

Reply:
I went back and forth on this one a couple times.

# Arguments For Leaving It In
1. There are some allocations that we skip if we know we don't care about errors.
2. It was like this before, so leaving it should mean no performance changes for current users.
3. It was like this before, so... mindiff?

# Arguments For Changing It
1. It simplifies the public API of the crate
2. It simplifies the implementation a bit

If these are always enabled, then I wonder if the signature for `InlinerBuilder::parse_and_inline_modules` should become:

```rust
impl InlinerBuilder {
    fn parse_and_inline_modules(&self, src_file: &Path) -> Result<InliningResult, ErrorKind> {}
}
```

The inner `InliningResult` would be the current `IncompleteInlining` struct, without the requirement that `errors` be non-empty. That approach has the advantage of being easier to reason over: "This function makes a best effort to inline modules into a syntax tree, returning an error if it can't produce _any_ syntax tree, and otherwise returning the best-effort tree and any problems it may have run into" but it makes it easier for someone to fail to handle incomplete inlining results, which seems bad.

Reply:
I like that!

Reply:
ErrorKind can probably be renamed to Error in that case too.

Reply:
Changes made.